app/models/planet.py

```diff
@@ -0,0 +1,27 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String, nullable=False)
+    distance_from_sun = db.Column(db.BigInteger, nullable=False)
```

Comment:
Depending on the unit represented by this integer an/or whether you intend for this to have no precision (i.e. be a float), bigint might be the best choice! 👍🏽